compiler/qsc_doc_gen/src/table_of_contents.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/// Returns the string of the contents of the generated top-level index.md file.
```

Comment:
I suggest we add a comment about where this is used. So that when we change the TOC and the test fails, we may assess the impact when updating the test.